packages/minifier-js/package.js

```diff
@@ -1,10 +1,10 @@
 Package.describe({
   summary: "JavaScript minifier",
-  version: "2.1.0"
+  version: "2.1.1"
```

Comment:
Only bumping the patch version of the `minifier-js` will make this functionality available to Meteor 1.5 apps (so you don't have to wait for 1.5.1). I think that's a good idea, since we probably would have shipped this functionality with 1.5 if it had been ready.